# Homogeneous coordinates : see e.g. p17 www.ensta-bretagne.fr/lebars/Obs_et_filt_de_Kalman.pdf
cmd_vel:
  topic: "/cmd_vel"
  angular_transform_matrix: [ 1,  0,  0,  0,
                              0,  1,  0,  0,
                              0,  0,  1,  0,
                              0,  0,  0,  1]
  linear_transform_matrix: [ 1,  0,  0,  0,
                             0,  1,  0,  0,
                             0,  0,  1,  0,
                             0,  0,  0,  1]
imu:
  topic: "/imu/data"
  # NWU to ENU
  angular_velocity_transform_matrix: [ 1,  0,  0,  0,
                                       0,  1,  0,  0,
                                       0,  0,  1,  0,
                                       0,  0,  0,  1]
  linear_acceleration_transform_matrix: [ 1,  0,  0,  0,
                                          0,  1,  0,  0,
                                          0,  0,  1,  0,
                                          0,  0,  0,  1]
  RPY_transform_matrix: [ 1,  0,  0,  0,
                          0,  1,  0,  0,
                          0,  0,  1,  1.5708,
                          0,  0,  0,  1]
fix:
  topic: "/navsat/fix"
vel:
  topic: "/vel"
  # NWU to ENU
  angular_transform_matrix: [ 1,  0,  0,  0,
                              0,  1,  0,  0,
                              0,  0,  1,  0,
                              0,  0,  0,  1]
  linear_transform_matrix: [ 0,  -1,  0,  0,
                             1,  0,  0,  0,
                             0,  0,  1,  0,
                             0,  0,  0,  1]
bOverrideFixStatus: true
max_linear_x_speed: 4.0
max_angular_z_speed: 4.0
szVectorNavInterfacePath: "192.168.0.16:5765"
rate: 50
